lib/utils/constants.dart

```diff
@@ -19,4 +19,8 @@ abstract class Constants {
   static const int textStyleGreen = 56;
   static const int textStyleBlue = 27;
   static const double textStyleOpacity = 2.0;
+
+  static const List<int> bodyList = [1, 2, 3, 4, 5, 6];
+  static const String body = "6";
```

Comment:
You should use a more representative name for this constant, and change the number for the sum of your initial dices values, so you keep consistency on the initial screen

Reply:
Fixed